src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -299,5 +315,42 @@ public static final class AutoThreeCargo {
       public static final SN_DoublePreference autoThreeCargoShooterRPM = new SN_DoublePreference(
           "autoThreeCargoShooterRPM", 3255);
     }
+
+    // auto 6
+    public static final class ThreeCargo {
+      public static final SN_DoublePreference shooterRPM1_6 = new SN_DoublePreference(
```

Comment:
the network tables id for each preference needs to be unique. shooterRPM1 is the first shooter RPM that will be used in the auto, but since mostly every auto will have a shooterRPM1 "_6" is added so they can be unique. I know that technically the variable name and network tables id don't have to match but I only see that leading to confusion and hidden bugs